### PR TITLE
FlutterFlow 4.1.85 (new cask)

### PR DESCRIPTION
**Pull request by paour**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by paour** [comment]
The only place I could find a version for this app is in the iOS store for the iPad version of the app. I'm pretty sure the Mac and iPad versions are in sync because older versions are not permitted to edit a project, but it does look strange.

Some internet pages claim the app auto-updates, but it really doesn't, which is one of the reasons I would really like to automate installing updates.

File: Casks/f/flutterflow.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask "flutterflow" do
+  version "4.1.85"
+  sha256 :no_check
+
+  url "https://storage.googleapis.com/flutterflow-downloads/macos/flutterflow-latest-macos.dmg", verified: "storage.googleapis.com/flutterflow-downloads/"
```
Comment (khipp) [inline Casks/f/flutterflow.rb:5]:
```suggestion
  sha256 "e72270ca3acb2e6a045a9bb6c6fb80125d3755735dbb7d8f1e6ecd91a06191d8"

  url "https://storage.googleapis.com/flutterflow-downloads/macos/flutterflow-#{version}-macos.zip",
      verified: "storage.googleapis.com/flutterflow-downloads/"
```

File: Casks/f/flutterflow.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  name "flutterflow"
```
Comment (khipp) [inline Casks/f/flutterflow.rb:6]:
```suggestion
  name "FlutterFlow"
```

File: Casks/f/flutterflow.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  desc "Build applications faster than ever"
```
Comment (khipp) [inline Casks/f/flutterflow.rb:7]:
```suggestion
  desc "Visual development platform"
```

We should describe what the app does or is (see [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc)).

File: Casks/f/flutterflow.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  homepage "https://flutterflow.io/"
+
+  # Documentation: https://docs.brew.sh/Brew-Livecheck
+  livecheck do
+    url "https://apps.apple.com/us/app/flutterflow-build-different/id6457995947"
+    strategy :page_match
+    regex(%r{<p class="l-column small-6 medium-12 whats-new__latest__version">Version (\d+\.\d+\.\d+)</p>})
+  end
```
Comment (khipp) [inline Casks/f/flutterflow.rb:15]:
```suggestion
  livecheck do
    url "https://storage.googleapis.com/flutterflow-downloads/macos/appcast.xml"
    strategy :sparkle
  end
```

File: Casks/f/flutterflow.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+  depends_on macos: ">= :catalina"
+
+  app "FlutterFlow.app"
+
+  zap trash: [
+    "~/Library/Application Support/io.flutterflow.prod.mac",
+    "~/Library/Caches/io.flutterflow.prod.mac",
+    "~/Library/Preferences/io.flutterflow.prod.mac.plist",
```
Comment (khipp) [inline Casks/f/flutterflow.rb:24]:
```suggestion
    "~/Library/Application Support/io.flutterflow.prod.mac",
    "~/Library/Caches/io.flutterflow.prod.mac",
    "~/Library/HTTPStorages/io.flutterflow.prod.mac",
    "~/Library/Preferences/io.flutterflow.prod.mac.plist",
    "~/Library/Saved Application State/io.flutterflow.prod.mac.savedState",
```

**Review by khipp — COMMENTED** [review]
Thank you for your submission, @paour!

I found an Appcast XML feed that we can use to fetch the latest version in the `livecheck` block.

On a side note: Please remove any auto-generated comments and format the file with `brew style --fix flutterflow.rb`.

File: Casks/f/flutterflow.rb (hunk 5 above)
Comment (paour) [reply Casks/f/flutterflow.rb:24]: Obviously the tool I use to detect preference and other files is not good enough, sorry.

File: Casks/f/flutterflow.rb (hunk 4 above)
Comment (paour) [reply Casks/f/flutterflow.rb:15]: I tried `brew find-appcast /Applications/FlutterFlow.app` but it didn't return anything. Is there a better way?

**Comment by paour** [comment]
> I found an Appcast XML feed that we can use to fetch the latest version in the `livecheck` block.

I tried `brew find-appcast /Applications/FlutterFlow.app` but it didn't return anything. Is there a better way?

> On a side note: Please remove any auto-generated comments and format the file with `brew style --fix flutterflow.rb`.

Sorry I keep missing comments. I did run `brew style --fix`, but it doesn't flag template comments unfortunately. Thanks for the reminder.

**Comment by khipp** [comment]
> I tried `brew find-appcast /Applications/FlutterFlow.app` but it didn't return anything. Is there a better way?

You can use [Proxyman](https://proxyman.io/) to log the HTTP(s) traffic from installed apps. The free version has some restrictions but is sufficient.

**Review by khipp — APPROVED** [review]
Thank you, @paour!

**Pull request merged**